nsiqcppstyle_exe.py

```diff
@@ -430,7 +425,7 @@ def GetActiveFilter(self):
     def GetFilterFile(self, filterfile):
         if not os.path.exists(filterfile):
             return None
-        f = file(filterfile, 'r')
+        f = open(filterfile, 'r')
```

Comment:
Oof. Note to self: figure out a way to use contextlib for closing the filterfile properly